src/backend/main.ts

```diff
@@ -616,35 +616,6 @@ async function runWineCommandOnGame(
   })
 }
 
-// Calls WineCFG or Winetricks. If is WineCFG, use the same binary as wine to launch it to dont update the prefix
-ipcMain.handle('callTool', async (event, { tool, exe, appName, runner }) => {
```

Comment:
I moved this into `tools.tsx` to stop adding more callbacks in main.ts, it's becoming too big and it feels like a big bag of everything